executor/src/test/java/org/k8loud/executor/action/kubernetes/DeleteResourceActionTest.java

```diff
@@ -0,0 +1,44 @@
+package org.k8loud.executor.action.kubernetes;
+
+import data.ExecutionExitCode;
+import data.ExecutionRS;
+import data.Params;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.k8loud.executor.action.Action;
+import org.k8loud.executor.exception.ActionException;
+import org.k8loud.executor.exception.KubernetesException;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class DeleteResourceActionTest extends BaseTest {
+    private static final String GRACE_PERIOD_SECONDS = "5";
+    private static final String GRACE_PERIOD_SECONDS_KEY = "gracePeriodSeconds";
+
+    @ParameterizedTest
+    @MethodSource
+    void testIfKubernetesServiceIsCalled(Params params) throws ActionException, KubernetesException {
+        // when
+        Action action = new DeleteResourceAction(params, kubernetesService);
+        ExecutionRS rs = action.perform();
+
+        // then
+        verify(kubernetesService, times(1)).deleteResource(eq(params.getRequiredParam(NAMESPACE_KEY)),
+                eq(params.getRequiredParam(RESOURCE_NAME_KEY)), eq(params.getRequiredParam(RESOURCE_TYPE_KEY)), any());
+        assertEquals(ExecutionExitCode.OK, rs.getExitCode());
```

Comment:
No message check?

Reply:
Same as `HorizontalScalingActionTest` - out of scope

Reply:
why? deleteResource return a message so it should be checked here. Same to `HorizontalScalingActionTest`

Reply:
> no message check. you can simply add sth like:
> 
> ```
> private static final String RESULT = "result"
> -----
> when(kubernetesService).scaleHorizontally(anyString(), anyString(), anyString(), anyLong()).thenReturn(RESULT)
> -----
> assertEquals(RESULT, rs.getResult())
> ```

Ok, I see
Checking the value of result is out of scope but we can check if it has been embeded in the response, adding this suggestion